browser/brave_vpn/win/brave_vpn_helper/brave_vpn_helper_utils.cc

```diff
@@ -91,4 +94,40 @@ std::wstring GetBraveVpnHelperServiceName() {
   return name;
 }
 
+std::wstring GetBraveVpnHelperRegistryStoragePath() {
+  switch (chrome::GetChannel()) {
+    case version_info::Channel::CANARY:
+      return L"Software\\BraveSoftware\\Brave\\Vpn\\HelperServiceNightly";
+    case version_info::Channel::DEV:
+      return L"Software\\BraveSoftware\\Brave\\Vpn\\HelperServiceDev";
+    case version_info::Channel::BETA:
+      return L"Software\\BraveSoftware\\Brave\\Vpn\\HelperServiceBeta";
+    case version_info::Channel::STABLE:
+      return L"Software\\BraveSoftware\\Brave\\Vpn\\HelperService";
+    default:
```

Comment:
`case Channel::UNKNOWN:` instead of `default:` everywhere.

Reply:
Fixed here and all other places.